docs/docs/development/setup/getting-started.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+:::info
+Throughout this documentation you will see commands like
+
+```sh
+sudo apt update
+```
+
+These commands should be run in a terminal such as Bash (Linux/MacOS/Docker), PowerShell, or Command Prompt (Windows).
+:::
+
+:::tip
+We recommend reading through the setup process before following it step by step, to ensure that you are happy with installing the required dependencies.
+:::
+
+## Environment Setup
+
+There are two ways to setup the ZMK development environment:
+
+- [Docker](/docs/development/setup/docker) ;
+- [Native](/docs/development/setup/native), i.e. using the operating system directly.
```

Comment:
```suggestion
There are two ways to setup the ZMK development environment: using [Docker](docker.md) or using a [native setup](native.mdx).
```
I'd say we can keep this small since the details are already elaborated on the following paragraphs.

(We also prefer file path links in the rest of the docs)

Reply:
Will change to file path links, but I think the bullet points break up the text and make it easier to read. Otherwise it's just a block of text in this section. Really, I'd prefer to break up the following paragraphs a bit to be more visually appealing, but can't think of a good way to do so.